### DISCUSSION TITLE
milvus启动报错的问题

### DISCUSSION THREAD

**Discussion by anhuaxiang** [milvus-io/milvus#23160] [Q&A and General discussion]
前提：同一个namespace下已经有一个自建的minio，如下所示
![image](https://user-images.githubusercontent.com/26645743/229041694-ad3757a7-ee63-4713-9afe-331b0b42908b.png)

部署milvus的命令为
`helm install miluvs-standalone milvus-4.0.13.tgz  --set cluster.enabled=false --set etcd.replicaCount=1 --set minio.mode=standalone --set pulsar.enabled=false  --set attu.enabled=true` 

milvus的相关信息为：
![image](https://user-images.githubusercontent.com/26645743/229042585-63263cb0-d40b-44f3-89d2-f2df9240fb90.png)

milvus的启动报错信息为：
![image](https://user-images.githubusercontent.com/26645743/229042047-fb385874-c065-40f2-9f24-caad64d23a74.png)

为什么出现这种情况呢？

**Comment by LoveEachDay** [comment]
Yes, this is a known issue. Try to move minio service into another namespace.

**Reply by anhuaxiang** [reply]
不在同一个namespace下是可以启动的
顺便问下，这个issue我没搜到，可以贴个链接嘛

**Reply by LoveEachDay** [reply]
This is because kubernetes automatically inject service related environment variable to milvus pods at the same namespaces. The `MINIO_PORT` will be injected which we're failed to check this environment to be a valid port number.
